### PR TITLE
Don't open profile if it's Autoplay

### LINKED ISSUE #8372: Clicking on player icon in result panel with autoplay shows blank user panel
Clicking on player icon in result panel with autoplay shows blank user panel

Screenshot_1
Screenshot_2

osu!lazer version: 2020.320.1

repricable on any song

### PR DESCRIPTION
**Pull request by uzervlad**
Closes #8372

### CODE REVIEW

File: osu.Game/Users/Drawables/DrawableAvatar.cs [hunk 1]
```diff
@@ -68,7 +68,7 @@ private void openProfile()
             if (!OpenOnClick.Value)
                 return;
 
-            if (user != null)
+            if (user != null && user.Id != 1)
```
Comment (peppy) [inline osu.Game/Users/Drawables/DrawableAvatar.cs]:
```suggestion
            if (user?.Id > 1)
```

**Review by peppy — CHANGES_REQUESTED** [review]

**Comment by peppy** [comment]
While this fix is okay, a better approach may be to not link in the first place, as to avoid showing an erroneous tooltip.

**Review by peppy — APPROVED** [review]

File: osu.Game/Users/Drawables/DrawableAvatar.cs (hunk 1 above)
Comment (Yash-HCL) [reply osu.Game/Users/Drawables/DrawableAvatar.cs]: Probably a reasonable comment would also help to understand why should be user Id > 1? If not then what is the fallback?

Comment (peppy) [reply osu.Game/Users/Drawables/DrawableAvatar.cs]: probably making `GuestUser` a static/readonly and referencing its ID directly.

**Pull request merged**